injective/assetlist.json

```diff
@@ -129,6 +129,35 @@
         }
       ]
     },
+
+    {
+      "description": "ClonEliteFund invests in Cosmos ecosystem projects, focusing on scalability, security, and interoperability to maximize returns.",
+      "denom_units": [
+        {
+          "denom": "ibc/695B1D16DE4D0FD293E6B79451640974080B59AA60942974C1CC906568DED795",
+          "exponent": 0
+        },
+        {
+          "denom": "CLON",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/695B1D16DE4D0FD293E6B79451640974080B59AA60942974C1CC906568DED795",
```

Comment:
Because this is an IBC asset, please define the `traces` so when know where it's from